lib/Mojolicious/Guides.pod

```diff
@@ -54,6 +54,21 @@ L<features|feature> are enabled, even if examples don't specifically mention it.
 Some modules, like L<Mojo::Base> and L<Mojolicious::Lite>, will enable them for
 you automatically, whenever they are used.
 
+=item Variable names
+
+Where possible, the documentation will try to use variable names in example code
+that describes whatever type of data should be passed through to or is returned
+from methods and functions. For instance C<$bytes> or C<$chars> to distinguish
+whether it is encoded bytes or decoded characters, C<$bool> to indicate whether
+a value only indicates true or false, the use of C<$c> to denote a
+L<Mojolicious::Controller> object, or C<$app> to denote the application object.
```

Comment:
Maybe say "denote the L<application|Mojolicious> object" or "denote the L<Mojolicious> application object"?

Reply:
not sure what you mean with the
>or "denote the L application object"